docs/product-installation-guide/src/main/asciidoc/controller-jws-install-proc.adoc

```diff
@@ -0,0 +1,70 @@
++
+WARNING: Ensure the names of the {PRODUCT} deployments you are copying do not conflict with your existing deployments in the Red Hat JBoss Web Server instance.
+. Remove the `.war` extensions from the `controller.war` folder.
+. Copy the contents of the `__TEMP_DIR__/rhdm-7.0-controller-jws/SecurityPolicy/` directory to `__JWS_HOME__/bin`. When asked to overwrite files, select *Yes*.
+. In the `__JWS_HOME__/standalone/deployments/` directory, create an empty file named `controller.war.dodeploy`. This file ensures that the standalone {KIE_SERVER} Controller is automatically deployed when the server starts.
+. Add the `kie-server` role and user to the `_JWS_HOME_/tomcat8/conf/tomcat-users.xml` file. In the following example,  `<USERNAME>` and `<PASSWORD>` is a user and password combination of your choice:
++
+[source]
+----
+<role rolename="kie-server"/>
+<user username="<USER_NAME>" password="<PASSWORD>" roles="kie-server"/>
+----
+
+. In the `_JWS_HOME_/tomcat8/bin` directory of the instance running {KIE_SERVER}, create a readable `setenv.sh` file with the following content:
++
+`CATALINA_OPTS="-Xmx1024m -Dorg.jbpm.server.ext.disabled=true -Dorg.jbpm.ui.server.ext.disabled=true -Dorg.jbpm.case.server.ext.disabled=true -Dorg.kie.server.controller.user=<CONTROLLER_USER> -Dorg.kie.server.controller.pwd=<CONTROLLER_PWD> -Dorg.kie.server.id=<KIE_SERVER_ID> -Dorg.kie.server.location=http://<HOST>:<PORT>/kie-server/services/rest/server -Dorg.kie.server.controller=http://<HOST>:<PORT>/controller/rest/controller"` 
+. In the `_JWS_HOME_/tomcat8/bin` directory of the instance running the standalone {KIE_SERVER} Controller, create a readable `setenv.sh` file with the following content:
++
+`-Dorg.kie.server.controller.user=<CONTROLLER_USER> 
+-Dorg.kie.server.controller.pwd=<CONTROLLER_PWD> 
+-Dorg.kie.server.id=<KIE_SERVER_ID>
+-Dorg.kie.server.location=http://<HOST>:<PORT>/kie-server/services/rest/server
+-Dorg.kie.server.controller=http://<HOST>:<PORT>/controller/rest/controller` 
++
+. To start the standalone {KIE_SERVER} Controller, enter one of the following commands in the `_JWS_HOME_/tomcat8/bin` directory:
++
+** On Linux or UNIX-based systems:
++
+[source,bash]
+----
+$ ./startup.sh
+----
+** On Windows:
++
+[source,bash]
+----
+startup.bat
+----
+. After a few minutes, review the the `_JWS_HOME_/tomcat8/logs` directory and correct any errors.
+. To verify that the Controller is working on Red Hat JBoss Web Server, enter the following command. In this command, replace `<CONTROLLER>` and `<CONTROLLER_PWD>` with the values in the `tomcat-users.xml` file. The output of this command provides information about the {KIE_SERVER} instance.
++
+[source]
+----
+curl -X GET "http://<HOST>:<PORT>/controller/rest/controller/management/servers" -H  "accept: application/xml" -u '<CONTROLLER>:<CONTROLLER_PWD>'
+----
+
+. In a web browser, enter the value of the `org.kie.server.controller` property, for example `http://localhost:8080/kie-wb/rest/controller`.
```

Comment:
Please delete this point, the controller functionality was checked in previous step.